CMakeLists.txt

```diff
@@ -108,6 +109,9 @@ if(BUILD_ALL OR BUILD_RECON_ONLY)
     # This module contains all things trigger scintillator
     add_subdirectory(TrigScint)
 
+    # This module contains trigger algorithms.
+    add_subdirectory(Trigger/Algo)
+
```

Comment:
Since this commit was made prior to Tracking, this will cause a rebase CONFLICT and will need to be resolved manually. (Easy to do, just keep both).